url-to-html/app.js

```diff
@@ -322,3 +356,15 @@ function formatBytes(bytes, decimals, binaryUnits) {
     var unitChanges = Math.floor(Math.log(bytes) / Math.log(unitMultiple));
     return parseFloat((bytes / Math.pow(unitMultiple, unitChanges)).toFixed(decimals || 0)) + ' ' + unitNames[unitChanges];
 }
+
+function isCBDDomain(hostname){
+    return /cbd.int$/.test(hostname) || 
+           /cbddev.xyz$/.test(hostname)
+}
+
+function abortNetworkUrlRequest(url){
+
+    return /\/socket\.io/.test(url) ||
+           /\app\/authorize\.html$/.test(url);
```

Comment:
should ou also abort logging of errors? that potentially caused problems?

Reply:
should be ok, because i reduce the network timeout to 15 seconds, before it was 0 which was like infinite.